tests/test_function_hasher.py

```diff
@@ -0,0 +1,39 @@
+import pytest
```

Comment:
Minor: in https://github.com/pycodehash/pycodehash/pull/24 the test files follow the directory structure in the package.

i.e. this file would be `test_hashing.py`